Website/Packages/datax-pipeline/src/modules/flowDefinition/flowHelpers.js

```diff
@@ -471,10 +475,17 @@ export function convertFlowToConfig(flow) {
         },
         outputs: sinkers,
         outputTemplates: outputTemplates,
-        rules: convertFlowToConfigRules(rules)
+        rules: convertFlowToConfigRules(rules),
+        batchList: batchList,
+        version: getTimeStamp()
     };
 }
 
+function getTimeStamp() {
+    let timeStamp = (new Date()).toISOString().slice(0, 19).replace(/-/g, "").replace(/:/g, "").replace("T", "");
```

Comment:
prettier would have converted your double quotes to single quotes for you. Please reopen all 16 files and click save for each (or Save All) to have all your files converted to consistent format configured for this package.

Reply:
no I didn't add this. it was probably added by accident when I merged with master. I removed this.